.gitignore

```diff
@@ -16,11 +16,17 @@ x64/
 [Bb]in/
 [Oo]bj/
 
+[Tt]ools/
+
 # Enable "build/" folder in the NuGet Packages folder since NuGet packages use it for MSBuild targets
 !packages/*/build/
 
 # Prevent accidental re-checkin of NuGet.exe
 NuGet.exe
+nuget.exe
+
+# Build tools log
+build/run/init-tools.log
```

Comment:
Can we file a bug to not have that around?  In general the build should never write to the source tree.